app/controllers/conferences_controller.rb

```diff
@@ -6,6 +6,9 @@ class ConferencesController < ApplicationController
 
   def index
     @current = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc)
+    if @current.count == 1
+      redirect_to "/conferences/#{@current.first.id}"
```

Comment:
@divyanshumehta  the conference should have a splash page and public access for anyone to visit the link